shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -49,6 +50,9 @@ module Shelley.Spec.Ledger.BaseTypes
     -- * STS Base
     Globals (..),
     ShelleyBase,
+
+    -- * Shelley era
+    Shelley,
```

Comment:
Maybe in the future, when there are multiple eras, it might make sense to move them to one or more separate modules.

Reply:
I agree. I mostly didn't do so here because I realised that I was getting caught up in trying to work out what the best structure for the future was, and then how this would entail restructuring the current code, and that I had instead offered to push a _quick_ PR :-D